WORKSPACE.bzlmod

```diff
@@ -1,6 +1,6 @@
 workspace(name = "intellij_with_bazel")
```

Comment:
> Why not keep both WORKSPACE files, at least for one release? That way we have a fallback in case something goes wrong.
Because then we will have to keep them in sync. If something goes wrong, we can just revert this PR, can't we?

Reply:
> If something goes wrong, we can just revert this PR, can't we?

Given the number of files this PR touches, it's likely that someone will merge a change that makes the revert difficult at some point. But, you've already done most of the work for this and it's likely only going to impact us working on the plugin, so I'm not going to block the PR.